inc/views/pluggable/pagination.php

```diff
@@ -128,6 +128,115 @@ public function filter_localization( $data ) {
 		return $data;
 	}
 
+	/**
+	 * Determine whether to show the Jump to page HTML input.
+	 * 
+	 * @return bool
+	 */
+	private function show_jump_to_page_input() {
+		return ( ! $this->has_infinite_scroll() && get_theme_mod( 'neve_enable_jump_to_pagination' ) ) ? true : false;
+	}
+
+	/**
+	 * Create jump to navigation inputs.
+	 * 
+	 * @return mixed $markup HTML to output on page.
+	 */
+	private function create_jump_to_html() {
+
+		// Value escaped once we confirm it's not empty, we never save this value to the DB.
+		$request = $_SERVER['REQUEST_URI'] ?? ''; //phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+		
+		if ( empty( $request ) ) {
+			return;
+		}
+
+		$request = esc_attr( $request );
+
+		global $wp_query;
+
+		$search_query = esc_attr( get_search_query() );
+		$search_input = ! empty( $search_query ) ? "<input id='s' type='hidden' value='$search_query' name='s' />" : '';
+
+		$button_text = apply_filters( 'neve_pagination_jump_button_text', __( 'Go', 'neve' ) );
+		$button_text = esc_attr( $button_text );
+
+		$max_num_pages = absint( $wp_query->max_num_pages );
+
+		$current_page = ! empty( get_query_var( 'paged' ) ) ? get_query_var( 'paged' ) : '';
+
+		$markup = <<<MARKUP
+		<div id="nv-pagination-jump">
+			<form action="$request" method="GET">
```

Comment:
All attributes must be escaped. Use `esc_attr` for attributes, `esc_url` for URLs.

Reply:
For this the `$request` is escaped higher up right after we check if it exists, do we still escape values we put in manually? i.e do we do `method="esc_attr('GET')"` ? and also the same for the div id even when those are hardcoded attributes?

Reply:
We don't need to escape hardcoded values like `method="GET"` but we need to escape all the variables when outputting the code. For example if you do `$request = <whatever_escape_function>($request);` and then output it in HTML, the phpcs will throw an error.